app/controllers/communities_controller.rb

```diff
@@ -8,21 +8,20 @@ class CommunitiesController < ApplicationController
   def index
     count_post_for_this_week
     @categories = Community::CATEGORIES
-    @communities = if params.has_key?(:category)
-                     Community.where(category: params[:category]).order(created_at: :desc).page(params[:page]).per 7
-                   else
-                     Community.order(created_at: :desc).page(params[:page]).per 7
-                   end
-    @random_category_communities = Community.where(category: @categories.sample).order('post_count_this_week desc').limit(5)
+    if(params.has_key?(:category))
+      @communities = Community.where(category: params[:category]).order(created_at: :desc).page(params[:page]).per 7
+    else
+      @communities = Community.order(created_at: :desc).page(params[:page]).per 7
+    end
+    @random_category_communities = Community.where(category: @categories.sample).order("post_count_this_week desc").limit(5)
   end
 
   def show
     @community = Community.find(params[:id])
     @posts = @community.posts.limit(20).sort_by { |p| p.score }.reverse
     @subscriber_count = @community.subscribers.count
     @subscribed = if account_signed_in?
-                    Subscription.where(community_id: @community.id,
-                                       account_id: current_account.id).any?
+                    Subscription.where(community_id: @community.id,account_id: current_account.id).any?
                   else
                     false
                   end
```

Comment:
the alignment is still off here.. check

Reply:
![image](https://user-images.githubusercontent.com/57782379/212522967-27e75862-287a-4238-87cf-a5479b1eb011.png)
Is this not a proper alignment or should I refactor it differently @iMagesh

Reply:
this is not a proper alignment @tusharravindran

Reply:
Resolved @iMagesh